v0/destinations/yahoo_dsp/util.js

```diff
@@ -0,0 +1,152 @@
+const qs = require("qs");
+const { generateJWTToken } = require("../../../util/jwtTokenGenerator");
+const { httpSend } = require("../../../adapters/network");
+const { CustomError } = require("../../util");
+
+const { ACCESS_TOKEN_CACHE_TTL } = require("./config.js");
+const Cache = require("../../util/cache");
+
+const accessTokenCache = new Cache(ACCESS_TOKEN_CACHE_TTL);
+
+const getUnixTimestamp = () => {
+  return Math.floor(Date.now() / 1000);
+};
+
+const poiLocationType = [
+  "includeChains",
+  "includeWoeids",
+  "includeCategories",
+  "includeGids",
+  "excludeChains",
+  "excludeWoeids",
+  "excludeCategories",
+  "excludeGids"
+];
+let listType;
+const includes = {};
+const excludes = {};
+
+const populateIncludes = (list, audienceType) => {
+  list.forEach(element => {
+    const keys = Object.keys(element);
+    keys.forEach(elementKey => {
+      if (poiLocationType.includes(elementKey)) {
+        listType = audienceType;
+      }
+    });
+    if (!listType) {
+      throw new CustomError(`Audience type ${audienceType} not provided`, 400);
```

Comment:
Since I am seeing this convention in error message
```javascript
throw new CustomError(`{destinationName} I am an exception message`, <statusCode>);
```
Can we follow it ?